facelessuser/coloraide-extras
# HCT and CAM16 improvements and additions

- Add CAM16 UCS JMh
- Use spline to verify HCT achromatic boundary
- Enable HDR range in HCT
- Add ∆E HCT for use in HCT chroma gamut mapping
- Improve results of HCT chroma gamut mapping for better tonal pallets
- Better tools to verify and tune HCT and JMh achromatic boundaries
- Fix HCT gamut mapping test
